package.json

```diff
@@ -8,6 +8,7 @@
     "@aws-sdk/client-s3": "^3.620.1",
     "@bull-board/api": "6.5.3",
     "@bull-board/express": "6.5.3",
+    "@cfaester/enzyme-adapter-react-18": "^0.8.0",
```

Comment:
Did you mean to add this dependency?

Reply:
yeah, when i went to `yarn install`, it was missing.  @drewbo added something that uses it a few months ago.